swarm/src/handler/one_shot.rs

```diff
@@ -232,8 +223,6 @@ where
 /// Configuration parameters for the `OneShotHandler`
 #[derive(Debug)]
 pub struct OneShotHandlerConfig {
-    /// Keep-alive timeout for idle connections.
-    pub keep_alive_timeout: Duration,
```

Comment:
Removing this field is a breaking change. Mind sending a deprecation PR for it?

Reply:
sure, opened #4680